modules/loader-utils/package.json

```diff
@@ -42,6 +42,7 @@
   "dependencies": {
     "@babel/runtime": "^7.3.1",
     "@loaders.gl/worker-utils": "3.0.0-alpha.21",
+    "@loaders.gl/loader-utils": "3.0.0-alpha.21",
```

Comment:
The module depends on itself. Use relative imports instead.
eg:
```
import {WorkerLoaderObject, LoaderObject} from './types';
```
instead of
```
import {WorkerLoaderObject, LoaderObject} from '@loaders.gl/loader-utils';
```